scripts/validate-ci

```diff
@@ -20,3 +20,6 @@ if [ -n "$DIRTY" ]; then
     git status
     exit 1
 fi
+
+echo Checking if released versions are not changed
+bash check-kdm-data.sh
```

Comment:
Will this relative path always work?

Reply:
What is "always" in this case, we are switching directories here https://github.com/rancher/kontainer-driver-metadata/pull/638/files#diff-82349173b59979fec746335f8ee568bdf3d2c657b4faacf218d2ceaa4f9135d5R4